app/services/representation_order_creator.rb

```diff
@@ -32,5 +33,35 @@ def call_cp_endpoint
     end
   end
 
-  attr_reader :defendant_id, :offences, :maat_reference, :defence_organisation
+  def sanitise_defence_organisation
+    sanitise_postcode if defence_organisation.dig(:organisation, :address, :postcode)
+    sanitise_contact_details
+  end
+
+  def sanitise_postcode
+    postcode = defence_organisation.dig(:organisation, :address, :postcode)
+    defence_organisation[:organisation][:address].delete(:postcode) unless CommonPlatform::PostcodeValidator.new(postcode: postcode).call
+  end
+
+  def sanitise_contact_details
+    @contact = defence_organisation.dig(:organisation, :contact)
+    return unless contact
+
+    sanitise_email_addresses
+    sanitise_phone_numbers
+  end
+
+  def sanitise_email_addresses
+    %i[primaryEmail secondaryEmail].each do |email|
+      defence_organisation[:organisation][:contact].delete(email) unless CommonPlatform::EmailValidator.new(email: contact.dig(email)).call
+    end
+  end
+
+  def sanitise_phone_numbers
+    %i[home work mobile fax].each do |number|
```

Comment:
```suggestion
    %i[home mobile].each do |number|
```
I believe this should suffice, as the [work](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/f0b400488aa612c9ecae2ba440bcfd5514c47974/lib/schemas/global/apiContactNumber.json#L11-L13)/[fax](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/f0b400488aa612c9ecae2ba440bcfd5514c47974/lib/schemas/global/apiContactNumber.json#L27-L30) numbers can be strings

Reply:
👍